### DISCUSSION TITLE
[ERROR] checkCFG mission File!

### DISCUSSION THREAD

**Discussion by acg001** [telekom-security/tpotce#871] [Q&A] [answered]
After rebooting the host, I am getting this when I check the status of tpot.

Jul 24 15:38:03 supportingprotection docker-compose[15124]: ewsposter              |  => [ERROR] checkCFG mission File! logfile = /data/mailoney/log/commands.log. Abort!

The docker object in the web admin also shows as "no carrier".

I have tried the steps detailed in this thread "https://github.com/telekom-security/tpotce/issues/514", but it's still the same. I have also tried running the update.sh. Still i get the same error. This only happened after i rebooted the host, and when i tried on other installation, it happens when i restarted the tpot service..

The version i am using is 20.06.2 ISO package.

**Comment by acg001** [comment, marked as answer]
The error was solved by touching a file indicated in the error, in this case, /data/mailoney/log/commands.log

**Comment by t3chn0m4g3** [comment]
I opened an [issue](https://github.com/telekom-security/ewsposter/issues/4).
